text-generation-inference/server/text_generation_server/jetstream_pt_support/models/mixtral_model_hf.py

```diff
@@ -0,0 +1,40 @@
+
+from jetstream_pt.third_party.mixtral import config as mixtral_config
+from jetstream_pt.third_party.mixtral.model import Transformer
+
+#.model_exportable import Transformer, model_args
```

Comment:
Looks like a leftover from an old version that was imported from gemma model that has the same comment.